bindings/java/src/org/sleuthkit/datamodel/blackboardutils/ArtifactsHelper.java

```diff
@@ -0,0 +1,177 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.sleuthkit.datamodel.blackboardutils;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.apache.commons.lang3.StringUtils;
+import org.sleuthkit.datamodel.AbstractFile;
+import org.sleuthkit.datamodel.Blackboard;
+import org.sleuthkit.datamodel.BlackboardArtifact;
+import org.sleuthkit.datamodel.BlackboardAttribute;
+import org.sleuthkit.datamodel.SleuthkitCase;
+import org.sleuthkit.datamodel.TskCoreException;
+
+
+/**
+ * Helper class to help module modules create artifacts.
+ *
+ * This class helps ingest modules create miscellaneous artifacts.
+ * 
+ */
+public final class ArtifactsHelper extends AbstractArtifactHelper {
```

Comment:
Can you run the netbeans source formatting tool on this file? Some indentation is off.